pygmt/src/surface.py

```diff
@@ -20,6 +20,7 @@
     I="spacing",
     R="region",
     G="outgrid",
+    T='tensor',
```

Comment:
I am unsure, whether `tensor` is a good alias for the `T` flag of `surface`. To be consistent with the upstream GMT documentation (https://docs.generic-mapping-tools.org/dev/surface.html#t), I feel `tension` (as suggested by @weiji14) or `tensionfactor` (no underscore because of two bridging consonants) makes for sense.

Reply:
Let's use `tension`.

```suggestion
    T="tension",
```

Reply:
OK! I will revise it.